src/core/background/furniture.js

```diff
@@ -10,6 +10,11 @@ import { isSpine, addSpine } from "./spine.js";
 import { isText, addText } from "./text.js";
 import { isParticles, addParticles } from "./particles.js";
 
+const setName = createFn => config => {
+    const item = createFn(config);
+    config.name && item.setName(config.name);
+};
+
 export const furnish = scene => () => {
     const configs = scene.context.theme.furniture || [];
```

Comment:
were you going to change this from furniture to items?

Reply:
just doing that in another PR. It's across all the configs so I thought I'd split it off.